src/lib/secrets_helper.ts

```diff
@@ -133,20 +82,46 @@ const formatSecrets = (result: OneGraphSecretsResponse | undefined) => {
   return newSecrets
 }
 
-type OneGraphPayload = { authlifyToken: string | undefined }
+interface RequestHeaders {
+  get(name: string): string | null
+}
+
+interface IncomingMessageHeaders {
```

Comment:
The value of this can be an array or string. I'd call this `IncomingHttpHeaders` to match the node builtin `http.IncomingHttpHeaders`, and add a comment to that effect. I'd not use the Node type directly though so that it can be used elsewhere.